### ISSUE TITLE
Charts timeout on the first load

### ISSUE THREAD

**Issue by smacker** [src-d/sourced-ui#121]
For me it took 3 refreshes and 15 minutes to get data for all charts.
Superset doesn't respond at all during generating data for the charts. (all pages just "loading" until requests to gitbase finish with success of timeout)

**Comment by se7entyse7en** [comment]
This issue it's very old, let's see if with latest version still has the same problem. It was the case with `gsc`, I actually didn't remember to have struggled that much with `gitbase` + default charts. Let's close this if it doesn't apply anymore or let's discuss about how we could improve this.

**Comment by se7entyse7en** [comment]
Okay I've just read that this should be fixed by closing #307.

**Comment by smacker** [comment]
yes. It's better now than at the time when the issue was opened but on a big dataset it would still timeout.
